src/js/rx/glossary.js

```diff
@@ -1,5 +1,5 @@
-const glossary = [
-  {
+module.exports = {
+  glossary: [{
```

Comment:
🤔 I think my original reason was to get the benefits of native `Array.prototype` methods, but that turned out not to be necessary after all.